src/matchers.rs

```diff
@@ -432,6 +433,63 @@ impl Match for BodyExactMatcher {
     }
 }
 
+#[derive(Debug)]
+/// Match part of the body of a request.
+///
+/// ### Example (string):
+/// ```rust
+/// use wiremock::{MockServer, Mock, ResponseTemplate};
+/// use wiremock::matchers::body_string_contains;
+///
+/// #[async_std::main]
+/// async fn main() {
+///     // Arrange
+///     let mock_server = MockServer::start().await;
+///
+///     Mock::given(body_string_contains("hello world"))
+///         .respond_with(ResponseTemplate::new(200))
+///         .mount(&mock_server)
+///         .await;
+///
+///     // Act
+///     let status = surf::post(&mock_server.uri())
+///         .body_string("this is a hello world example!".into())
+///         .await
+///         .unwrap()
+///         .status();
```

Comment:
Probably worth adding an example with a JSON payload as well

Reply:
I'm not sure if it's worth adding a JSON example, this implementation just checks for a substring, the format doesn't really matter.